### ISSUE TITLE
Useless parts to delete

### ISSUE THREAD

**Issue by kYroL01** [kYroL01/ConvNet#6]
Maybe the lines
https://github.com/kYroL01/ConvNet/blob/master/my_alexnet_cnn.py#L211
must be delete, and int the lines
https://github.com/kYroL01/ConvNet/blob/master/my_alexnet_cnn.py#L218
run the sess with `logits` that return `train logits` is completely useless

**Comment by kYroL01** [comment]
Fixed by https://github.com/kYroL01/ConvNet/commit/be6bc3a1f52ae30d36e5e962beb183b6da8de1eb
